src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -108,6 +108,18 @@
      */
     protected abstract LocalInspectionTool getInspectionTool();
 
+    protected void sendUsageReport() {
+        String packageType = getProjectPackageType() + "-deps";
+        try {
+            Utils.sendUsageReport(packageType);
+        } catch (IOException e) {
+            getLog().debug("Usage report failed: " + ExceptionUtils.getRootCauseMessage(e));
```

Comment:
Since this debug message is duplicated twice in the code, we can move it into the ```Utils.sendUsageReport``` method.